modules/labs-react/common/spec/useThemeRTL.spec.ts

```diff
@@ -165,7 +164,7 @@ describe('styles', () => {
         const {result} = renderHook(() => useThemeRTL());
         const cssProp = result.current.themeRTL(style);
 
-        expect((cssProp['&.hover'] as CSSObject).paddingRight).toEqual(expectedPadding);
+        expect((cssProp['&:hover'] as CSSObject).paddingRight).toEqual(expectedPadding);
```

Comment:
same here, the test would fail even though wallaby said that it was correct

Reply:
See comment above.